### DISCUSSION TITLE
Is there a way to get the zip file version?

### DISCUSSION THREAD

**Discussion by AlexSmalexx98** [LMMS/lmms#6301] [General]
I cannot use installers on my Windows 10, so would anyone please give me just the actual folder that the installer installs to your computer?

Thanks, AlexSmalexx98

**Comment by RiedleroD** [comment]
You can choose the destination folder in the installer yourself. If some antivirus blocks the executable from running at all, disable that. If you need administrator rights to start the installer… why? That shouldn't be the case. Admin rights should only be required when installing to a system directory.

**Reply by AlexSmalexx98** [reply]
Thanks, usually exe's do run but for some unknown reason every time I try to open the installer, windows just keeps telling me it couldn't open it. This always happens with installers and installers only and I have zero idea why. I was just wondering if I could get **the actual program or folder** instead of having to use the installer for it, since mine doesn't work. Thanks though!

**Reply by RiedleroD** [reply]
perhaps you're choosing the wrong installer? Try the 32-bit ones and see if the nightly version works better than the stable one.

regarding the request about the folder: I can't extract it myself, I'm on linux, sorry.

**Reply by AlexSmalexx98** [reply]
I've tried those too and they don't seem to work either. Not sure how to handle this situation.

It's alright, thanks for attempting to help though.

**Reply by qnebra** [reply]
Wasn't Windows preventing you by showing warning? It is known thing that UAC didn't like unsigned installers.

**Reply by allejok96** [reply]
Portable 1.2.2:
https://portableapps.com/apps/music_video/lmms-portable

Or here is a zip of it:
https://drive.google.com/file/d/1Tp5JWLM_JaYiCTbqYpMu43-xhl8AJ7m8/view?usp=sharing

But chances are if the first link doesn't work you won't be able to run anything downloaded from the internet. Maybe it's Windows 10 S or some school computer with restrictions?

**Reply by AlexSmalexx98** [reply]
Yo thanks so much man! Seems to be working great!

**Reply by allejok96** [reply]
Out of curiosity: did the first link work?

**Reply by AlexSmalexx98** [reply]
Very sorry for getting to you late, dunno why I didn't mention this earlier, but yes it did work, although when I used it all of the instruments sounded very raspy and robotic sounding, so I tried the second one and the same thing happened-- but here's the weirder part, both of them suddenly stopped playing audio whenever I tried loading soundfonts or just any sound at all. I've tried using every audio input/output, I've tried seeing if the volume was off on my computer and the application, I tried re-downloading it, but to no avail. Again, sorry for getting back to you later than I would've wanted, but hopefully you see this and can find a way to fix it. Thanks!

**Comment by tipico-games** [comment]
This proposal aims to streamline file organization by mirroring `include/` and `src/` structures, improving clarity and maintainability. Key aspects:

1. **Folder Structure**: Aligning `include/` and `src/` for consistent navigation.
2. **Class Namespacing**: Introducing clear namespaces for better code organization.
3. **Presentation**: Collapsible folder groups are suggested to simplify file navigation compared to raw tree output.

Feedback is requested on both the structural approach and the usability of grouped views.

**Comment by headquarter8302** [comment]
For future reference, some archival programs like 7zip are able to open the .exe file and show the raw contents of the archive inside the executable.

Open the executable in 7zip and extract everything in the file to a directory of your choice. This will bypass registering LMMS on Windows as an application, so mind that if you want to uninstall/organize it

**Comment by tresf** [comment]
https://gist.github.com/tresf/6a3daf8987162e7e57dcf058c3affda5
